src/generated/sdk.ts

```diff
@@ -6683,6 +6683,14 @@ export function getSdk(client: GraphQLClient, withWrapper: SdkFunctionWrapper =
     },
     verifyEmail(variables: VerifyEmailMutationVariables): Promise<VerifyEmailMutation> {
       return withWrapper(() => client.request<VerifyEmailMutation>(print(VerifyEmailDocument), variables));
+    },
+    checkToken(variables: {token: any}): Promise<any> {
+      return withWrapper(() => client.request<any>(print(gql`
+            query checkToken($token: String!) {
+        checkToken(token: $token) {
+          isValid
+        }
+      }`), variables))
```

Comment:
I'm waiting for a https://github.com/UserOfficeProject/user-office-frontend/pull/44 then this will be properly generated, currently the generate command fails

Reply:
Merged it! 🦾